app/templates/frameworks/contract_review.html

```diff
@@ -0,0 +1,117 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Review your contract – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+      {
+        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
+        "label": "Your " + framework.name + " application"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    {%
+      with
+      heading = "Check the details you’ve given before returning the signature page for {}".format(supplier_framework.declaration.nameOfOrganisation),
```

Comment:
Should be `heading = "... {} ..." | format(supplier_framework.declaration.nameOfOrganisation)` to avoid unicode issues